### PR TITLE
Implement `purge` command

### PR DESCRIPTION
**Pull request by Integralist**

### CODE REVIEW

File: pkg/app/run_test.go [hunk 1]
```diff
@@ -213,6 +214,20 @@ COMMANDS
     List Fastly datacenters
 
 
+  purge [<flags>]
+    Remove object(s) from the Fastly cache
```
Comment (Integralist) [inline pkg/app/run_test.go:218]: Invalidate objects in the Fastly cache.

File: pkg/app/run_test.go [hunk 2]
```diff
@@ -213,6 +214,20 @@ COMMANDS
...
+
+        --all                    Purge everything from a service
+        --file=FILE              Purge a service with a line separated list of
+                                 Surrogate Keys
+        --key=KEY                Purge a service of items tagged with a
```
Comment (Integralist) [inline pkg/app/run_test.go:223]: replace items with objects

File: pkg/app/run_test.go [hunk 3]
```diff
@@ -213,6 +214,20 @@ COMMANDS
     List Fastly datacenters
 
 
+  purge [<flags>]
+    Remove object(s) from the Fastly cache
+
+        --all                    Purge everything from a service
+        --file=FILE              Purge a service with a line separated list of
```
Comment (Integralist) [inline pkg/app/run_test.go:221]: Purge a service of a newline delimited list of Surrogate Keys.

File: pkg/app/run_test.go [hunk 4]
```diff
@@ -213,6 +214,20 @@ COMMANDS
...
+                                 Surrogate Keys
+        --key=KEY                Purge a service of items tagged with a
+                                 Surrogate Key
+    -s, --service-id=SERVICE-ID  Service ID (falls back to FASTLY_SERVICE_ID,
+                                 then fastly.toml)
+        --soft                   A 'soft' purge marks the affected object as
```
Comment (Integralist) [inline pkg/app/run_test.go:227]: make plural: objects

Comment (Integralist) [reply pkg/app/run_test.go:227]: marks affected objects

Comment (Integralist) [reply pkg/app/run_test.go:227]: Add logic below to error if --soft used along with --url

**Review by triblondon — APPROVED** [review]
reviewed together on zoom

**Pull request merged**